.travis.yml

```diff
@@ -1,4 +1,5 @@
-sudo: false
+sudo: required
+dist: trusty
```

Comment:
According to this you might not need sudo anymore:
https://docs.travis-ci.com/user/trusty-ci-environment/